tests/test_logging.py

```diff
@@ -0,0 +1,153 @@
+import datetime
+import logging
+import unittest
+
+from core_lib.helpers.func_utils import Keyable
+from core_lib.helpers.logging import Logging
+
+
+class User(Keyable):
+    def __init__(self, u_name, password):
+        self.u_name = u_name
+        self.password = password
+
+    def key(self) -> str:
+        return f'User(u_name:{self.u_name}, password:{type(self.password).__name__})'

```

Comment:
remove the password from the print
reason:
we are printing the type of password which is str. it feels redundant.